pkg/stanza/docs/operators/regex_parser.md

```diff
@@ -8,17 +8,18 @@ This operator makes use of [Go regular expression](https://github.com/google/re2
+|--------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `id`         | `regex_parser`   | A unique identifier for the operator.                                                                                                                                                                                                 |
+| `output`     | Next in pipeline | The connected operator(s) that will receive all outbound entries.                                                                                                                                                                     |
+| `regex`      | required         | A [Go regular expression](https://github.com/google/re2/wiki/Syntax). The named capture groups will be extracted as fields in the parsed body.                                                                                        |
+| `parse_from` | `body`           | The [field](../types/field.md) from which the value will be parsed.                                                                                                                                                                   |
+| `parse_to`   | `attributes`     | The [field](../types/field.md) to which the value will be parsed.                                                                                                                                                                     |
+| `on_error`   | `send`           | The behavior of the operator if it encounters an error. See [on_error](../types/on_error.md).                                                                                                                                         |
+| `if`         |                  | An [expression](../types/expression.md) that, when set, will be evaluated to determine whether this operator should be used for the given entry. This allows you to do easy conditional parsing without branching logic with routers. |
+| `timestamp`  | `nil`            | An optional [timestamp](../types/timestamp.md) block which will parse a timestamp field before passing the entry to the output operator.                                                                                              |
+| `severity`   | `nil`            | An optional [severity](../types/severity.md) block which will parse a severity field before passing the entry to the output operator.                                                                                                 |
+| `cache`      | `nil`            | An optional [cache](../types/cache.md) block which will enable a memory cache for parsing the same input repeatedly.                                                                                                                  |
```

Comment:
Since the cache is only used in this operator, let's keep the information within this document. We can always extract it later if it becomes shared between multiple operators.